### PR TITLE
[10.0][mig] purchase_delivery_split_date

### PR DESCRIPTION
**Pull request by LoisRForgeFlow**
Changes done in this migration:
* Basic migration actions (odoo tags, manifest...)
* Fix data used in the test.

### CODE REVIEW

**Comment by LoisRForgeFlow** [comment]
cc @eficent

File: purchase_delivery_split_date/__manifest__.py [hunk 1]
```diff
@@ -0,0 +1,17 @@
+# -*- coding: utf-8 -*-
+# © 2014-2016 Numérigraphe SARL
+# © 2016 Eficent Business and IT Consulting Services, S.L.
```
Comment (MiquelRForgeFlow) [inline purchase_delivery_split_date/__manifest__.py]: If you want, you can put 2017 😁

**Review by MiquelRForgeFlow — APPROVED** [review]
LGTM :+1:

File: purchase_delivery_split_date/README.rst [hunk 2]
```diff
@@ -0,0 +1,48 @@
+.. image:: https://img.shields.io/badge/licence-AGPL--3-blue.svg
+   :target: http://www.gnu.org/licenses/agpl-3.0-standalone.html
+   :alt: License: AGPL-3
+
+============================================================
+Split Purchase Deliveries in one reception per expected date
+============================================================
+
+When this module is installed, each Purchase Order you confirm will
+generate one Incoming Shipment for each expected date indicated in the
```
Comment (AaronHForgeFlow) [inline purchase_delivery_split_date/README.rst]: I'd use the word schedule date instead of expected because is what the user sees on the form view.

**Review by AaronHForgeFlow — APPROVED** [review]
:+1: Tested on runbot + code review :lollipop:

**Review by JordiBForgeFlow — APPROVED** [review]

**Pull request merged**